src/components/forms/bouquet/BouquetOwnerForm.vue

```diff
@@ -34,47 +90,74 @@ watch(choice, () => {
 
 <template>
   <div>
-    <label class="fr-label" for="owner"
-      >Choisissez si vous souhaitez gérer ce {{ topicsName }}&nbsp;:</label
+    <DsfrRadioButtonSet
+      :required="true"
+      v-model="choice"
+      :options="radioOptions"
+      :legend="`Choisissez si vous souhaitez gérer ce ${topicsName}&nbsp;:`"
+    />
+    <fieldset
+      v-if="choice === 'organization'"
+      class="fr-fieldset organizations"
     >
-    <fieldset id="owner" class="fr-fieldset">
-      <div class="fr-fieldset__content" role="radiogroup">
-        <DsfrRadioButton
-          v-model="choice"
-          name="owner"
-          value="owner"
-          label="En votre propre nom"
+      <div class="fr-fieldset__element">
+        <DsfrSelect
+          v-model="selectedOwnOrganization"
+          label="Organisations dont vous faites partie."
+          defaultUnselectedText="Sélectionnez une ogranisation"
+          id="ownerOrg"
+          :options="selectOptions"
+          @update:modelValue="onSelectOwnOrganization()"
         />
-        <div>
-          <DsfrRadioButton
-            v-model="choice"
-            :disabled="organizations.length === 0"
-            name="organization"
-            value="organization"
-            label="En tant qu'organisation"
-          />
-          <div v-if="choice === 'organization'">
-            <select
-              class="fr-select"
-              @change="
-                onSelectOrganization(($event.target as HTMLInputElement)?.value)
-              "
-            >
-              <option selected disabled value="">
-                Choisissez une organisation
-              </option>
-              <option
-                v-for="(org, idx) in organizations"
-                :key="idx"
-                :selected="topic.organization?.id === org.id"
-                :value="idx"
-              >
-                {{ org.name }}
-              </option>
-            </select>
-          </div>
-        </div>
+      </div>
+      <div v-if="userStore.isAdmin" class="fr-fieldset__element">
+        <label class="fr-mt-2v" for="any-org-select-bouquet"
+          >Cherchez une autre organisation.</label
+        >
+        <Multiselect
+          id="any-org-select-bouquet"
+          ref="multiselect"
+          v-model="selectedAnyOrganization"
+          :options="options"
+          track-by="id"
+          placeholder="Ex: Lyon Métropole"
```

Comment:
```suggestion
          placeholder="Ex : Ministère de la Transition Ecologique"
```

Plus cohérent avec notre public :-)